x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -207,7 +207,7 @@ export const ElasticsearchOverview = () => {
             'ingestData',
             codeSnippetArguments
           )}
-          showTryInConsole={getConsoleRequest('ingestData')}
+          consoleRequest={getConsoleRequest('ingestData')}
```

Comment:
I think this maybe was an oversight of https://github.com/elastic/kibana/pull/164766?

Reply:
yes this is fixed in `main`